sympy/matrices/tests/test_matrices.py

```diff
@@ -2283,24 +2283,38 @@ def test_LDLsolve():
     b = A*x
     soln = A.LDLsolve(b)
     assert soln == x
+
     A = Matrix([[0, -1, 2],
                 [5, 10, 7],
                 [8,  3, 4]])
     x = Matrix(3, 1, [-1, 2, 5])
     b = A*x
     soln = A.LDLsolve(b)
     assert soln == x
+
     A = Matrix(((9, 3*I), (-3*I, 5)))
     x = Matrix((-2, 1))
     b = A*x
     soln = A.LDLsolve(b)
     assert expand_mul(soln) == x
+
     A = Matrix(((9*I, 3), (-3 + I, 5)))
     x = Matrix((2 + 3*I, -1))
     b = A*x
-    soln = A.cholesky_solve(b)
+    soln = A.LDLsolve(b)
```

Comment:
Do you know why cholesky solve was used here? Why have you changed it?

(I'm not disagreeing with this, just asking because I don't understand)

Reply:
It looks like merged in #14474 with no discussion.
And actually, LDL and cholesky is not much different.